### PR TITLE
Fix Fib Fountain Hole 

### PR DESCRIPTION
**Pull request by BaziForYou**
just fixed this hole
image

### CODE REVIEW

File: gtav/base.lua [hunk 1]
```diff
@@ -29,6 +29,11 @@ Citizen.CreateThread(function()
     
     -- FIB lobby: 105.4557, -745.4835, 44.7548
     RequestIpl("FIBlobby")
+
+	--Fib Fountain 174.184,-667.902,43.140
```
Comment (MichaelCoding25) [inline gtav/base.lua:33]: hey, you might wanna turn this line into:
```lua
-- FIB Fountain 174.184, -667.902, 43.140
```

Comment (BaziForYou) [reply gtav/base.lua]: Done :D

**Review by MichaelCoding25 — APPROVED** [review]
Tested and seems to work

**Comment by Bob74** [comment]
@BaziForYou Thanks for fixing this, I totally missed it.
@MichaelCoding25 Thanks for testing!

**Pull request merged**